src/Modals/ModalBody.tsx

```diff
@@ -1,19 +1,13 @@
 import { ReactNode } from 'react';
 import styled, { withTheme } from 'styled-components';
-import { BaseTheme } from '../Theme';
 
 interface ModalBodyProps {
   /**
    * the content of the Modal Body
    */
   children: ReactNode;
-  /**
-   * The MarkOneTheme
-   */
-  theme: BaseTheme;
```

Comment:
Should `theme: BaseTheme` be replaced with `theme: DefaultTheme` like in the `Table` components in these cases where `theme: BaseTheme` is being removed?

Reply:
Aha. Actually those `Table` components were incorrect. I went back and removed all of those in ded04e2, plus another instance in the `GlobalCSS` component.